common/src/main/java/io/pravega/example/tensorflow/Recognition.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.tensorflow;
+
+/**
+ * An immutable result returned by a recognizer describing what was recognized.
+ */
+public final class Recognition {
+    /**
+     * A unique identifier for what has been recognized. Specific to the class, not the instance of
+     * the object.
+     */
+    private final Integer id;
+    private final String title;
+    private final Float confidence;
+    private BoxPosition location;
```

Comment:
If this class is immutable, why is this not final?

Reply:
There is a ```setLocation(BoxPosition location) {...}" here. So, I commented it and made it a final